_maps/map_files/stations/cerestation.dmm

```diff
@@ -29986,9 +29986,9 @@
 /area/station/maintenance/fore2)
 "ezn" = (
 /obj/machinery/camera{
-	c_tag = "AI Satellite Service Bay";
 	dir = 8;
-	network = list("SS13","MiniSat")
+	network = list("SS13","MiniSat");
```

Comment:
The PTL is not a part of the minisat on the Farragus. Honestly, all these cameras could probably make due with just being on the "SS13" network.